### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by mardahl**
Added a clarification on the CSV that is not clear from the article that is linked, as I have had customers ask about this - it simply needed to be easier and clearer to understand it seems..

Also added information about using AAD Connect to provision the target tenant user accounts.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -350,11 +350,20 @@ T2Tbatch                   Syncing ExchangeRemoteMove 1
 ```
 
 > [!NOTE]
-> The email address in the CSV file must be the one specified in the target tenant, not the source tenant.
+> The email address in the CSV file must be the one specified in the target tenant (e.g. userA@targettenant.onmicrosoft.com), not the source tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
> The email address in the CSV file must be the one specified in the target tenant (for example, userA@targettenant.onmicrosoft.com), not the one in the source tenant.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 2]
```diff
@@ -350,11 +350,20 @@ T2Tbatch                   Syncing ExchangeRemoteMove 1
...
 >
 > [For more information on the cmdlet click here](/powershell/module/exchange/new-migrationbatch)
 >
-> [For an example CSV file click here](/exchange/csv-files-for-mailbox-migration-exchange-2013-help)
+> [For some example CSV file info click here](/exchange/csv-files-for-mailbox-migration-exchange-2013-help)
+
+Here is a minimal example CSV.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
The following is a minimal example CSV file:
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 3]
```diff
@@ -543,6 +552,15 @@ These conversions happen automatically during the migration process. No manual s
 
 This can be done before the migration is complete, but you should not assign a license prior to stamping the _ExchangeGuid_ attribute or the conversion of MailUser object to mailbox will fail and a new mailbox will be created instead. To mitigate this risk, it is best to wait until after the migration is complete, and assign licenses during the 30 day grace period.
 
+### Can I use Azure AD Connect to sync users to the new tenant if I am keeping the on-prem Active Directory?
+
+Yes. It is possible to have two instances of Azure AD Connect syncronize to different tenants.
+However there are some things you need to be aware of.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
Yes. It is possible to have two instances of Azure AD Connect synchronize to different tenants.
However, there are some things you need to be aware of.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 4]
```diff
@@ -543,6 +552,15 @@ These conversions happen automatically during the migration process. No manual s
...
+
+- Preprovisioning the users accounts with the script provided in this articale should not be done. Instead a selective OU sync of the users in scrope for the migation can be performed to populate the target tenant - you will receive a warning about the UPN not matching during AAD Connect configuration.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
- Preprovisioning the user's accounts with the script provided in this article should not be done. Instead, a selective OU sync of the users in scope for the migration can be performed to populate the target tenant; you will receive a warning about the UPN not matching during Azure AD Connect configuration.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 5]
```diff
@@ -543,6 +552,15 @@ These conversions happen automatically during the migration process. No manual s
...
+- Depending on your current state of Hybrid Exchange, you will need to verify that the on-prem directory objects have the required attributes populated correctly (i.e. msExchMailboxGUID, proxyAddresses etc.) before attempting to sync to another tenant, or you will run into issues with double mailboxes and migration failures.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
- Depending on your current state of Hybrid Exchange, you need to verify that the on-prem directory objects have the required attributes (such as msExchMailboxGUID and proxyAddresses) populated correctly before attempting to sync to another tenant, or you will run into issues with double mailboxes and migration failures.
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 6]
```diff
@@ -543,6 +552,15 @@ These conversions happen automatically during the migration process. No manual s
...
+- You will need to take some ekstra steps to manage UPN transitioning. Changing it on-prem once the migration has completed for a user, unless you are also moving the custom domain during a cut-over migration.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
- You need to take some extra steps to manage UPN transitioning, changing it on-prem once the migration has been completed for a user unless you are also moving the custom domain during a cut-over migration.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mardahl, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by mardahl** [comment]
Thank for the proofing @JohanFreelancer9.
Always a pleasure with you on the commit <3

**Comment by mardahl** [comment]
#ATCP

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**